app/routes/profile.rb

```diff
@@ -0,0 +1,16 @@
+module ExercismWeb
+  module Routes
+    class Profile < Core
+      get '/profiles/:username/:public_profile_key' do |username, key|
```

Comment:
Since we're in `/profiles/` we don't need `profile` in the last segment. `/profiles/:username/:public_key` should be enough.